Docker/Image/Builder/ImageBuilder.php

```diff
@@ -208,22 +169,17 @@ public function getCommands()
         return $this->commands;
     }
 
-
     /**
      * @param array $commands
-     * @return $this
      */
-    public function setCommands(array $commands)
+    private function setCommands(array $commands)
```

Comment:
Protoze se to vola jen v constructoru, gettery primo se nekontrolujou, ale spravny nastaveni se kontroluje treba tady https://github.com/keboola/docker-bundle/blob/master/Tests/Docker/ImageBuilderTest.php#L128